e2e-pw/src/oss/poms/action-row/tagger.ts

```diff
@@ -0,0 +1,34 @@
+import { Locator, Page } from "src/oss/fixtures";
+
+type TaggerMode = "sample" | "label";
+
+export class TaggerPom {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  async clickTagger() {
+    this.page.getByTestId("action-tag-sample-labels").click();
+  }
```

Comment:
Should this be in `grid-actions-row` or `modal-actions-row` (new file) as `openTagger()` / `closeTagger()`?